Doc/library/concurrent.futures.rst

```diff
@@ -63,10 +63,17 @@ Executor Objects
        using a large value for *chunksize* can significantly improve
        performance compared to the default size of 1.  With
        :class:`ThreadPoolExecutor`, *chunksize* has no effect.
+       
+       By default, a reasonable number of tasks are
+       queued beyond the number of workers, an explicit *prefetch* count may be
+       provided to specify how many extra tasks should be queued.
```

Comment:
The documentation for chunksize uses the phrasing "this method chops iterables into a number of chunks which it submits to the pool as separate tasks", and since not all executors even use chunks (ThreadPoolExecutor ignores the argument), I figured I'd stick with "tasks". It does kind of leave out a term to describe a single work item; the docs uses chunks and tasks as synonyms, with no term for a single work item.